### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by cap10morgan**
This is the cleaned-up version of what I did a few Fridays ago to spike this out.

It implements these 2 cards:
1. [Pivotal card](https://www.pivotaltracker.com/story/show/99177178)
1. [Pivotal card](https://www.pivotaltracker.com/story/show/99422054)

Tests pass (there aren't a ton yet, though), and I hit the 2 responders running in docker-compose from a REPL connected to the same Rabbit broker and they worked as expected.

### CODE REVIEW

**Comment by cap10morgan** [comment]
README updates incoming shortly.

**Comment by cap10morgan** [comment]
@tank157 OK, should be ready for review.

File: src/voter_registration_works/handlers.clj [hunk 1]
```diff
@@ -1,8 +1,127 @@
...
+(def voter-register-coercer (sc/coercer voter-register-schema
+                                        state-coercion))
+
+(defn voter-register
+  "A handler for registering people to vote."
+  [message]
+  (log/debug "Received voter.register message:" (pr-str message))
+  (let [coerced-register-request (coerce-or-throw voter-register-coercer message)]
+    {:status :ok
+     :next (states/voter-register (:state coerced-register-request)
```
Comment (tank157) [inline src/voter_registration_works/handlers.clj]: I know you're trying not to be US centric in the data structure above, but then you go right to US centric with `(states/voter-register ...)`. Not a blocker, just something to consider.

Comment (cap10morgan) [inline src/voter_registration_works/handlers.clj]: Yes, but that's intentional. Someday if you supported other countries I imagine you'd have a branch point before this that called fns in a namespace other than `states`. And then we might move things around to make it clearer that we're talking about _U.S._ states.

But really, too much non-US-centrism in here is just YAGNI at this stage. So if anything I'd want to err on the side of not going too far with it.

**Comment by tank157** [comment]
Other than what was mentioned above, and which isn't a blocker, looks good to me! Feel free to Deliver if you're not going to make any changes based on it.

**Pull request merged**